steering_council_membership.md

```diff
@@ -3,11 +3,15 @@
 The Pangeo steering council members are:
 
 - Ryan Abernathey (@rabernat)
+- Tom Augspurger (@TomAugspurger)
+- Anderson Banihirwe (@andersy005)
+- Aimee Barciauskas (@abarciauskas-bgse)
+- Anne Fouilloux (@annefou)
+- Chelle Gentemann (@cgentemann)
```

Comment:
Chelle has already been on the SC...we just didn't update this page.

Reply:
The list was reordered slightly but she was in fact already on the list.